client-src/modules/ansi-html/LICENSE

```diff
@@ -0,0 +1,202 @@
+Apache License
```

Comment:
We use MIT 😕 , would it be a problem?

Reply:
Yes, ansi-html was published under Apache 2.0. I cannot simply re-license code that isn't mine. Apache 2.0 is more restrictive than MIT, but it doesn't stop us from copying and adapting code, as long as the license is preserved.

Reply:
We can't accept this, sorry

Reply:
WE need rewrite code in this case on our solution

Reply:
In that case it might be far easier to just change the dependency from `ansi-html` to `ansi-html-community`. Since the original author of `ansi-html` is not reacting, someone else created a new npm package with the same code plus the fix.

Reply:
Yes, we can do it too